elytron/src/test/resources/org/wildfly/extension/elytron/elytron-transformers-4.0-reject.xml

```diff
@@ -0,0 +1,2 @@
+<subsystem xmlns="urn:wildfly:elytron:5.0">
```

Comment:
given this file is named `elytron-transformers-4.0-reject.xml`, shouldn't it be keeping the `urn:wildfly:elytron:4.0`namespace?

Reply:
No this is testing transformation to 4 not from 4 - by the end this will likely end up having version 6 of the namespace.